modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/util/JournalUtil.java

```diff
@@ -234,6 +238,21 @@ public static long getPreviewPlid(
 				LayoutConstants.DEFAULT_PARENT_LAYOUT_ID);
 		}
 
+		PermissionChecker permissionChecker =
+			PermissionCheckerFactoryUtil.create(themeDisplay.getUser());
```

Comment:
I think we can using themeDisplay.getPermissionChecker() instead of

Reply:
Your idea is ok. I will update this.